x/staking/handler.go

```diff
@@ -0,0 +1,60 @@
+package staking
+
+import (
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/staking"
+	"github.com/desmos-labs/juno/parse/client"
+	"github.com/desmos-labs/juno/parse/worker"
+	"github.com/desmos-labs/juno/types"
+	"github.com/forbole/bdjuno/database"
+	"github.com/rs/zerolog/log"
+
+	"fmt"
+)
+
+func MsgHandler(tx types.Tx, index int, msg sdk.Msg, w worker.Worker) error {
+	log.Info().Str("tx_hash", tx.TxHash).Int("msg_index", index).
+		Str("msg_type", msg.Type()).Msg("found message")
+
+	if len(tx.Logs) == 0 {
+		log.Info().Str("tx_hash", tx.TxHash).Int("msg_index", index).
+			Msg("skipping message as it was not successful")
+		return nil
+	}
+
+	db, ok := w.Db.(database.BigDipperDb)
+	if !ok {
+		return fmt.Errorf("given database instance is not a BigDipperDb")
+	}
+
+	timestamp, err := time.Parse("2006-01-02T15:04:05Z", tx.Timestamp)
```

Comment:
This time format can be simplified by using 

```suggestion
	timestamp, err := time.Parse(time.RFC3339, tx.Timestamp)
```